lib/shortcuts/api.md

```diff
@@ -2,35 +2,93 @@
 
 ### Table of Contents
 
--   [Lambda][1]
-    -   [Parameters][2]
-    -   [Properties][3]
-    -   [Examples][4]
--   [ScheduledLambda][5]
-    -   [Parameters][6]
-    -   [Examples][7]
--   [QueueLambda][8]
-    -   [Parameters][9]
-    -   [Examples][10]
--   [StreamLambda][11]
-    -   [Parameters][12]
-    -   [Examples][13]
--   [ServiceRole][14]
-    -   [Parameters][15]
-    -   [Properties][16]
+-   [LambdaOptions][1]
```

Comment:
Unfortunately this markdown file generated from `documentationjs` has become a lot less readable because `documentationjs` does not currently handle `constructor`-level parameter annotations.

If we want it to, we should submit a fix for https://github.com/documentationjs/documentation/issues/1091